onebusaway-android/build.gradle

```diff
@@ -39,19 +40,30 @@ repositories {
         // CUTR Releases
         url "https://github.com/CUTR-at-USF/cutr-mvn-repo/raw/master/snapshots"
     }
+
+    maven {
+        // for testlib
+        url  "http://dl.bintray.com/acrowntest/test"
+    }
 }
 
 android {
     compileSdkVersion 23
     buildToolsVersion "25.0.2"
 
     defaultConfig {
-        minSdkVersion 9
+        minSdkVersion 15
```

Comment:
Given that we're hiding/showing social features already based on a Region API field, I think we could probably extend this same check to being API Level 15 and higher, and retain support for API Level 9?  Is there anything else that's blocking API Level 9 from being supported?